addons/src/Makefile.am

```diff
@@ -1,31 +1,4 @@
+include $(top_srcdir)/build/vars.init.mk
 include $(top_srcdir)/build/vars.build.mk
-
-geanyplugins_LTLIBRARIES = addons.la
-
-addons_la_SOURCES = \
-	addons.h \
-	ao_blanklines.h \
-	ao_doclist.h \
-	ao_openuri.h \
-	ao_systray.h \
-	ao_bookmarklist.h \
-	ao_markword.h \
-	ao_tasks.h \
-	ao_xmltagging.h \
-	ao_wrapwords.h \
-	ao_copyfilepath.h \
-	addons.c \
-	ao_blanklines.c \
-	ao_doclist.c \
-	ao_openuri.c \
-	ao_systray.c \
-	ao_bookmarklist.c \
-	ao_markword.c \
-	ao_tasks.c \
-	ao_xmltagging.c \
-	ao_wrapwords.c \
-	ao_copyfilepath.c
-
-addons_la_LIBADD = $(COMMONLIBS)
-
 include $(top_srcdir)/build/cppcheck.mk
```

Comment:
This is not acceptable, `make check` has to behave the same everywhere -- and especially, it has to run the checks when called from the top-level as it used to, so it's run by e.g. `make distcheck`.  AFAICS, it doesn't.

I guess a possible solution would be to "simply" populate a variable like i.e. `CPPCHECK_DIRS` and have that in the top-level Makefile's cppcheck rule instead of `$(srcdir)` or something.